README.md

```diff
@@ -1,15 +1,170 @@
-- Join the [chat](https://community.getdbt.com/) on Slack for live discussions and support
-- Find [dbt events](https://events.getdbt.com) near you
-- Check out [the blog](https://blog.getdbt.com/) for the latest news on dbt's development and best practices
+# 🤔 Who is the target user of this dbt package?
+- You use more than one of Fivetran' app platform connectors, including:
+    - [Apple App Store](https://fivetran.com/docs/applications/apple-app-store)
+    - [Google Play](https://fivetran.com/docs/applications/google-play)
+- You use dbt
+- You want a modeling layer that cleans, tests, and prepares your app platform data for analysis as well as leverage the analysis ready models outlined above.
+
+# 🎯 How do I use the dbt package?
+To effectively install this package and leverage the pre-made models, you will follow the below steps:
+## Step 1: Pre-Requisites
+You will need to ensure you have the following before leveraging the dbt package.
+- **Connector**: Have all relevant Fivetran app platform connectors syncing data into your warehouse. 
+- **Database support**: This package has been tested on **BigQuery**, **Snowflake**, **Redshift**, **Postgres** and **Databricks**. Ensure you are using one of these supported databases.
+- **dbt Version**: This dbt package requires you have a functional dbt project that utilizes a dbt version within the respective range `>=1.0.0, <2.0.0`.
+
+## Step 2: Installing the Package
+Include the following github package version in your `packages.yml`
+> Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+```yaml
+packages:
+  - package: fivetran/app_reporting
+    version: [">=0.1.0", "<0.2.0"]
+```
+## Step 3: Configure Your Variables
+### Database and Schema Variables
+By default, this package looks for your app platform data in your target database. If this is not where your advertising data is stored, add the relevant `_database` variables to your `dbt_project.yml` file (see below).
+
+By default, this package also looks for specific schemas from each of your connectors. The schemas from each connector are highlighted in the code snippet below. If your data is stored in a different schema, add the relevant `_schema` variables to your `dbt_project.yml` file:
+
+```yml
+vars:
+  apple_store_schema: itunes_connect
+  apple_store_database: your_database_name
+  
+  google_play_schema: google_play
+  google_play_database: your_database_name 
+```
+
+### Connector Selection
+The package assumes that all connector models are enabled, so it will look to pull data from all of the connectors listed above. If you don't want to use certain connectors, disable those connectors' models in this package by setting the relevant variables to `false`.
+
+```yml
+vars:
+  app_reporting__google_play_enabled: false
+  app_reporting__apple_store_enabled: false
+```
```

Comment:
Is there a reason someone would use this package if they were only using one of the packages?

Reply:
Great question, I was kind of thinking that this could scale out to more app platforms - like perhaps.. Windows App store...Amazon App store (if we ever get there).

Reply:
I love the future thinking here! Since there currently are only two connectors available to work with this package, I would say we should remove these configs to reduce the length and complexity of the README. 

Down the road, when we implement additional connectors 🤞  we can incorporate these details to the README.